src/beautifiers/eslint.coffee

```diff
@@ -14,21 +15,35 @@ module.exports = class ESLintFixer extends Beautifier
   }
 
   beautify: (text, language, options) ->
-    return new @Promise((resolve, reject) ->
+    return new @Promise((resolve, reject) =>
       editor = atom.workspace.getActiveTextEditor()
       filePath = editor.getPath()
       projectPath = atom.project.relativizePath(filePath)[0]
 
       result = null
-      allowUnsafeNewFunction ->
+      allowUnsafeNewFunction =>
         importPath = Path.join(projectPath, 'node_modules', 'eslint')
+        eslint = null
         try
-          CLIEngine = require(importPath).CLIEngine
-
-          cli = new CLIEngine(fix: true, cwd: projectPath)
-          result = cli.executeOnText(text).results[0]
-
-          resolve result.output
-        catch err
-          reject(err)
+          eslint = requireg('eslint')
```

Comment:
This order is incorrect.
We want:
1. Project's locally installed ESLint
2. Globally installed ESLint
3. Atom-Beautify's fallback ESLint

As @UziTech [pointed out](https://github.com/Glavin001/atom-beautify/pull/2147#discussion_r192610452), `requireg` is not actually what we want.

I think https://github.com/sindresorhus/import-global is what we want.

```coffeescript
try
  importPath = Path.join(projectPath, 'node_modules', 'eslint')
  eslint = require(importPath) # Local ESLint for project
catch
  try
    importGlobal = require('import-global');
    eslint = importGlobal('eslint') # Globally installed ESLint
  catch
    eslint = require('eslint') # ESLint packaged with Atom-Beautify
```

Reply:
@zhuyingda `import-global` also did not work? Wow.

Although idea is using `requireg` and manually calling the internal `nodePathResolve` function: https://github.com/h2non/requireg/blob/master/lib/resolvers.js#L37-L54

```javascript
const nodePathResolve = require("requireg/lib/resolvers")[1];
const globalPackage = nodePathResolve(module, "eslint");
```

Until @UziTech 's PR is merged: https://github.com/h2non/requireg/pull/15

Reply:
https://github.com/h2non/requireg/pull/15 is merged in [requireg v0.2.0](https://www.npmjs.com/package/requireg)

so `eslint = requireg('eslint', true)` should work to install globally

Reply:
@zhuyingda Looks like this should be changed:

```suggestion
          eslint = requireg('eslint', true)
```